client/lib/url/is-external.ts

```diff
@@ -11,10 +11,16 @@ import { isLegacyRoute } from 'lib/route/legacy-routes';
 
 // Base URL used for URL parsing. The WHATWG URL API doesn't support relative
 // URLs, so we always need to provide a base of some sort.
-const BASE_HOSTNAME = 'base.invalid';
+const BASE_HOSTNAME = '__domain__.invalid';
```

Comment:
How is this change related to the fix?

Reply:
It's not, it must have slipped in from another branch. I'll remove it.

Reply:
Done.